### PR TITLE
Splits clothing.dm in 7 other files

### PR DESCRIPTION
**Pull request by lzimann**
Having every clothing type in the same file is pretty bad if you want to locate something quickly. I've moved to their respective subfolders.

### CODE REVIEW

File: code/modules/clothing/gloves/gloves.dm [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	equip_delay_other = 40
+
+
+/obj/item/clothing/gloves/worn_overlays(isinhands = FALSE)
+	. = list()
+	if(!isinhands)
+		if(damaged_clothes)
+			. += mutable_appearance('icons/effects/item_damage.dmi', "damagedgloves")
+		if(blood_DNA)
+			. += mutable_appearance('icons/effects/blood.dmi', "bloodyhands")
```
Comment (ninjanomnom) [inline code/modules/clothing/gloves/gloves.dm]: Does this proc really need to get copy pasted per subtype

File: code/modules/clothing/masks/masks.dm [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+	icon = 'icons/obj/clothing/masks.dmi'
+	body_parts_covered = HEAD
+	slot_flags = SLOT_MASK
+	strip_delay = 40
+	equip_delay_other = 40
+	var/mask_adjusted = 0
+	var/adjusted_flags = null
+
+
+/obj/item/clothing/mask/worn_overlays(isinhands = FALSE)
```
Comment (lzimann) [inline code/modules/clothing/masks/masks.dm]: lol this proc

File: code/modules/clothing/gloves/gloves.dm (hunk 1 above)
Comment (Supermichael777) [reply code/modules/clothing/gloves/gloves.dm]: no but they will need vars for those overlays

**Comment by Cyberboss** [comment]
`Repository` is for github docs and such

File: code/modules/clothing/gloves/gloves.dm (hunk 1 above)
Comment (ninjanomnom) [reply code/modules/clothing/gloves/gloves.dm]: It was a rhetorical question

**Review by ShizCalev — DISMISSED** [review]
Rename the files to _*.dm to keep them consistent with https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#files, as a few of them have the same names as other files in the repo.

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**